include/zephyr/net/wifi_mgmt.h

```diff
@@ -866,7 +876,19 @@ struct wifi_mgmt_ops {
 	 * @return 0 if ok, < 0 if error
 	 */
 	int (*set_rts_threshold)(const struct device *dev, unsigned int rts_threshold);
-
+	/** Set max inactivity time for STA in AP mode
+	 *
+	 * If a station does not send anything in max_inactivity_time seconds,
+	 * AP may decide to throw away the STA. Possibly because of STA is no more or
+	 * has move out of the range.
+	 *
+	 * @param dev Pointer to the device structure for the driver instance.
+	 * @param max_inactivity_time Maximum inactivity time in second.
+	 *
+	 * @return 0 if ok, < 0 if error
+	 */
+	int (*ap_sta_max_inactivity)(const struct device *dev,
```

Comment:
Adding separate APIs for each param isn't scalable, we should have a single API e.g., `ap_sta_config` or `ap_config` and then add few param's, WDYT?

Reply:
Added single API to handle AP mode parameter .configuration